Slack-Apps/homework/responses/1.1-bolt-intro.md

```diff
@@ -1,4 +1,6 @@
-# Introduction to Bolt JS
+## Week 1: Introduction to Bolt JS
```

Comment:
Don't put the "Week 1:" there because the algorithm automatically adds the "Week 1:" part. It would show up as "Week 1: Week 1: Introduction to Bolt JS" if you keep it this way.